sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -56,4 +56,18 @@ export class ApplicationApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async confirmationOfAssessment(applicationId: number): Promise<any> {
```

Comment:
IMO this method should not return any data. The success or error could be expressed as HTTP codes.